src/main/java/mate/academy/hw4/myArrayList/SimpleArray.java

```diff
@@ -0,0 +1,58 @@
+package mate.academy.hw4.myArrayList;
+
+import java.util.Iterator;
+
+public class SimpleArray<E> implements Simple<E> {
+    private E[] values;
+
+    public SimpleArray() {
+        values = (E[]) new Object[0];
+    }
+
+    @Override
+    public boolean add(E e) {
+        try {
+            E[] temp = values;
+            values = (E[]) new Object[temp.length + 1];
+            System.arraycopy(temp, 0, values, 0, temp.length);
+            values[values.length - 1] = e;
+            return true;
+        } catch (ClassCastException ex) {
+            ex.printStackTrace();
+        }
+        return false;
+    }
+
+    @Override
+    public void delete(int index) {
+        try {
+            E[] temp = values;
+            values = (E[]) new Object[temp.length - 1];
+            System.arraycopy(temp, 0, values, 0, index);
+            int amountEmemAfterIndex = temp.length - index - 1;
+            System.arraycopy(temp, index + 1, values, index, amountEmemAfterIndex);
+        } catch (ClassCastException ex) {
+            ex.printStackTrace();
```

Comment:
do not printStackTrace in Util classes, any exceptions should be handled by user of class

Reply:
ok